stations.csv

```diff
@@ -70491,4 +70491,5 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74632;Brescia Violino;brescia-violino;;;45.54553750281283;10.168658833685743;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8317451;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74632;
 74633;Minusio;minusio;;;46.17404650703084;8.819574802968333;;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8505417;t;;f;;f;;f;8505417;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74633;
 74634;Catania (Piazza Risorgimento);catania-piazza-risorgimento;;;37.50155099221305;15.068981718421918;22160;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74634;
-74635;Monsampolo del T;monsampolo-del-t;;;42.881404;13.803619;19616;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S07023;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74635;
\ No newline at end of file
+74635;Monsampolo del T;monsampolo-del-t;;;42.881404;13.803619;19616;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S07023;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74635;
+74636;Lascari;lascari;;;38.011695;13.928291;19906;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S12070;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74636;
```

Comment:
is_suggestable is enabled ✅ should probably be set to `false` if we only have realtime ID for this station

Reply:
done